src/components/InputTextbox/index.jsx

```diff
@@ -0,0 +1,15 @@
+export default function index(props) {
+
+  return (
+    <div>
+      <input
+        type='text'
+        onChange={props.onChange}
+        value={props.onChange}
```

Comment:
I believe the value prop here does is redundant and doesn’t actually do anything. Especially given that the value is pointing to a function.